src/configure.h

```diff
@@ -135,3 +135,14 @@
 
 
 //#define USE_COUNTER_SIGNATURES
+
+//
+//   Define which COSE objects can be decoded
+//
+
+#define COSE_DECODE_ENCRYPT
```

Comment:
I am worried about two things on this
1) There is an assumption that linkers will not pull in functions that are not referenced.  This is true for some linkers, but I am not sure how universal it is.  
2) Should there be some additional build versions to deal with this.  Specifically, does there need to be some changes to the test functions to look at this variables as well.